syncapi/storage/postgres/current_room_state_table.go

```diff
@@ -223,17 +223,22 @@ func (s *currentRoomStateStatements) upsertRoomState(
 		_, containsURL = content["url"]
 	}
 
+	headeredJSON, err := json.Marshal(event)
+	if err != nil {
+		return err
+	}
+
 	// upsert state event
 	stmt := common.TxStmt(txn, s.upsertRoomStateStmt)
-	_, err := stmt.ExecContext(
+	_, err = stmt.ExecContext(
 		ctx,
 		event.RoomID(),
 		event.EventID(),
 		event.Type(),
 		event.Sender(),
 		containsURL,
 		*event.StateKey(),
-		event.JSON(),
+		headeredJSON,
```

Comment:
For claritys sake, I would like the column name to be renamed to something like this instead of its current `event_json` which is now ambiguous and has incorrect SQL-level comments. Please update.

Reply:
Have renamed the columns to `headered_event_json` and updated the SQL-level comments.